packages/babel-plugin-transform-unicode-property-regex/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@babel/plugin-transform-unicode-property-regex",
+  "version": "3.0.0",
+  "description": "Compile Unicode property escapes in Unicode regular expressions to ES5.",
+  "homepage": "https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-unicode-property-regex",
+  "main": "dist/index.js",
```

Comment:
we are using `lib/index.js` in our packages https://github.com/babel/babel/blob/a75e69ececb83bdd7964e6d90e1257ca9cc00430/packages/babel-plugin-transform-async-generator-functions/package.json#L7